umbraco/Umbraco-CMS
# V8/Warn editor when deleting a media item that is in use

### Prerequisites


If there's an existing issue for this PR then this fixes #8693

### Description
This PR introduces a warning to editors when they try to delete a media item or folder in the media section when the item or one of the underlying items of a folder is in use.

![delete-warning](https://user-images.githubusercontent.com/1193822/95472135-f4a84900-0982-11eb-8091-576b60232005.gif)


### Note for reviewers
This PR builds on top of PR #9116 so it's recommend that that one is reviewed and merged first.

### Steps to test

See steps for media in #9116. When you try to delete the item a warning is shown + information of the references.
